lib/index.js

```diff
@@ -47,10 +47,10 @@ function NwBuilder(options) {
         argv: process.argv.slice(2)
     };
     // Intercept the platforms and check for the legacy platforms of 'osx' and 'win' and
-    // replace with 'osx32', 'osx64', and 'win32', 'win64' respectively.
+    // replace with 'osx64', and 'win32', 'win64' respectively.
     if(typeof options.platforms != 'undefined'){
         if(options.platforms.indexOf('osx') >= 0){
-            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx32', 'osx64');
+            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx64');
```

Comment:
I mean, the reason I would feel confident about making the changes that are left is because NW.js itself has dropped support for the platform: `osx32` cannot work if you upgrade NW.js. If they want nw-builder to work with older versions of NW.js, they can update their build scripts to pass in the right `--platform` flag. But still totally up to you.